### PR TITLE
Improved parsing custom label template inside dialog.

### PR DESCRIPTION
**Pull request by michal-damiecki**
CW-147

### CODE REVIEW

**Comment by michal-damiecki** [comment]
screenshot from 2018-03-21 14-39-20

**Comment by michal-damiecki** [comment]
screenshot from 2018-03-21 16-00-33

File: src/schema/SchemaActions.js [hunk 1]
```diff
@@ -84,18 +84,8 @@ export function toLocalSchema(schema, state, parentProperty, uiSchema = {}) {
...
-              separator,
-              schemaId
-            ] = uiSchema['ui:customLabel'].match(/(?!%)(.*?)(?=%)/g);
-            const [
-              resourceSchemaId,
-              resourceProperty
-            ] = resource.split('.');
-
-            const parents = getSelectedParents(state, schemaId, resourceSchemaId)
+          if (uiSchema['ui:lebelTemplate'] && uiSchema['ui:requiredResource'] && data.length > 0) {
```
Comment (sealionkat) [inline src/schema/SchemaActions.js]: `ui:labelTemplate` :slightly_smiling_face:

File: src/schema/SchemaActions.js [hunk 2]
```diff
@@ -140,7 +130,15 @@ export function toLocalSchema(schema, state, parentProperty, uiSchema = {}) {
               request$.subscribe(response => {
                 data.forEach((value, index) => {
                   enumValues.push(value.id);
-                  const label = `${response[index].payload[resourceProperty]}${separator}${value.name}`;
+                  const label = parseLabelTemplate(
+                    uiSchema['ui:lebelTemplate'],
```
Comment (sealionkat) [inline src/schema/SchemaActions.js]: `ui:labelTemplate` :slightly_smiling_face:

**Comment by thayashi** [comment]
LGTM, other than the "lebel -> label "

**Comment by michal-damiecki** [comment]
Fixed

**Comment by thayashi** [comment]
LGTM

**Comment by thayashi** [comment]
@michal-damiecki could you please create a new tag for gohan_webui as 2.22.0?

**Pull request merged**